http.go

```diff
@@ -1,14 +1,23 @@
 package main
 
 import (
+	"fmt"
 	"log"
 	"net/http"
+	"os"
 	"sync"
 	"time"
 
 	auth "github.com/heroku/lumbermill/Godeps/_workspace/src/github.com/heroku/authenticater"
+	influx "github.com/heroku/lumbermill/Godeps/_workspace/src/github.com/influxdb/influxdb-go"
 )
 
+var influxDbStaleTimeout = 24 * time.Minute // Would be nice to make this smaller, but it lags due to continuous queries.
+var influxDbSeriesCheckPrefixes = []string{
+	"MaxMean1mLoad.10m.dyno.dyno.load.",
```

Comment:
I don't think it's an existing use case, to be quite honest. For now though, I'll put the queries in the array with a "%s" for the token. That'll at least be something closer to a more general approach.